### PR TITLE
KT-18515 Intention to apply De Morgan laws to all / any / none calls

### PR DESCRIPTION
**Pull request by t-kameyama**
https://youtrack.jetbrains.com/issue/KT-18515
https://youtrack.jetbrains.com/issue/KT-10034

### CODE REVIEW

**Review by dimonchik0036 — CHANGES_REQUESTED** [review]
Could you please transfer the changes to [intellij-kotlin](https://github.com/JetBrains/intellij-kotlin)?

File: idea/resources/META-INF/inspections.xml [hunk 1]
```diff
@@ -834,6 +834,66 @@
...
+    <intentionAction>
+      <className>org.jetbrains.kotlin.idea.intentions.ConvertTakeIfToTakeUnlessIntention</className>
+      <category>Kotlin</category>
+    </intentionAction>
+
+    <intentionAction>
+      <className>org.jetbrains.kotlin.idea.intentions.ConvertTakeUnlessToTakeIfIntention</className>
+      <category>Kotlin</category>
+    </intentionAction>
+
```
[the remark this answers is not in the document]
Comment (t-kameyama) [reply idea/resources/META-INF/inspections.xml:896]: I'd like to suggest two quick fixes for one call expression.
```kotlin
fun test(list: List<Int>) {
    // suggest "Replace 'all' with 'none'" and "Replace 'all' with 'any'"
    list.all/*caret*/ { it == 1 } 
}
```
Is it possible to suggest two quick fixes in one Intention class?  Or should I create Inspection with INFORMATION level?

**Comment by t-kameyama** [comment]
> Could you please transfer the changes to [intellij-kotlin](https://github.com/JetBrains/intellij-kotlin)?

https://github.com/JetBrains/intellij-kotlin/pull/35

File: idea/resources/META-INF/inspections.xml (hunk 1 above)
[the remark this answers is not in the document]
Comment (dimonchik0036) [reply idea/resources/META-INF/inspections.xml:896]: Oh, I got it. In this case, it is better to leave it as it is (many intentions).

**Comment by dimonchik0036** [comment]
Thanks! I will continue the review in JetBrains/intellij-kotlin#35.

**Pull request closed without merging**